DOCS/man/mpv.rst

```diff
@@ -1200,9 +1200,17 @@ PROTOCOLS
 
     Digital TV via DVB. (Linux only.)
 
-``mf://[filemask|@listfile]`` ``--mf-...``
-
-    Play a series of images as video.
+``mf://[@listfile|filemask|glob|printf-format]`` ``--mf-...``
+
+    Play a series of images as a video. If the URL begins with ``@``, it is
+    interpreted as a sequence of paths separated by newlines. If the URL
```

Comment:
I don't think the *URL* is interpreted as that. Rather it reads a file at that path and interprets *its contents*.
No?